e2e/compose-env.yaml

```diff
@@ -17,5 +17,14 @@ services:
        - notary-fixtures:/fixtures
       command: ['notary-server', '-config=/fixtures/notary-config.json']
 
+  evil-notary-server:
```

Comment:
why is it evil?

Reply:
See https://github.com/moby/moby/pull/36515/files#diff-4b1e56bb77ac16f2ccf956fc24cf0a82L350 :stuck_out_tongue: 
Definitely open to name it something else, but I'll need to re-generate the certs :sweat_smile: